src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -111,13 +122,13 @@ export module DotNet {
 
   /**
    * Creates a JavaScript object reference that can be passed to .NET via interop calls.
-   * 
+   *
    * @param jsObject The JavaScript Object used to create the JavaScript object reference.
    * @returns The JavaScript object reference (this will be the same instance as the given object).
    * @throws Error if the given value is not an Object.
    */
   export function createJSObjectReference(jsObject: any): any {
-    if (jsObject instanceof Object) {
+    if (jsObject && typeof jsObject === 'object') {
       if (!jsObject.hasOwnProperty(JSObject.ID_KEY) || cachedJSObjectsById[nextJsObjectId] !== jsObject) {
```

Comment:
Currently this line throws when `jsObject` is an ES6 module instance, because it doesn't have the `hasOwnProperty` function.

However, as per [my comments here](https://github.com/dotnet/aspnetcore/pull/25028/files#r475707409), I don't think we should be doing this stuff with `JSObject.ID_KEY` anyway, so this problem should go away on its own if that was fixed.